pages/infrastructure-as-code-library/index.html

```diff
@@ -1,12 +1,13 @@
 ---
 layout: default
-title: Gruntwork Library
+title: Infrastructure as Code Library
 excerpt: A collection of reusable, battle-tested, production-ready infrastructure code.
```

Comment:
Looking at the Infra as Code Library page, the underlying tech's are sort of hidden away. I'm thinking that perhaps under the "Infra as Code Library Features" and above the "Infra Packages" sections, we add another section as follows?

> **Built with Popular DevOps Tooling and Languages**
> Our code is primarily written in:
> - Terraform
> - Packer
> - Go
> - Bash
> - Python

Reply:
It's the very first feature listed on that page: "Infrastructure as Code. Written in Terraform, Go, Python, and Bash."

Reply:
I know, but we could really use the logos in a big and bold way on this page. I'll draft something up when I have a free moment.

Reply:
As a reminder, the thing customers respond most favorably too, by far, is seeing the code. There are already 9 logos you have to scroll through before you see what code is available. I don't think we should make them any bigger. It's just noise before getting to the thing the customer cares about: "oooh, you already have ZooKeeper code... and ECS, wow!"

Reply:
Alright, maybe it's low priority since we already have the logos in the bottom right of each package anyway. If I feel inspired I'll take a swing at it at some point, keeping your concern in mind.

Reply:
Alright, maybe it's low priority since we already have the logos in the bottom right of each package anyway. If I feel inspired I'll take a swing at it at some point, keeping your concern in mind.